pkg/system/mknod_windows.go

```diff
@@ -3,10 +3,10 @@
 package system
 
 func Mknod(path string, mode uint32, dev int) error {
-	// should not be called on cli code path
+	// should not be called on the Windows path
```

Comment:
I agree on both. It's still very tempting for me to be verbose in comments, but that said, it wasn't my comment :) I've removed it.